app/adapters/application.js

```diff
@@ -0,0 +1,4 @@
+import JSONAPIAdapter from '@ember-data/adapter/json-api';
+
+export default class ApplicationAdapter extends JSONAPIAdapter {
```

Comment:
Adding the adapter and serializer removes the deprecation message from the browser console.